horovod/run/common/util/settings.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Uber Technologies, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+class Settings(object):
+
+    def __init__(self, verbose=False):
```

Comment:
Since you're adding this over-arching settings object, are there any other parameters we could fold into it?  Could be useful to limit brittleness of data transmission over cmdline args.